Client/WebFilters/AdblockerType.swift

```diff
@@ -8,56 +8,55 @@ import Shared
 private let log = Logger.browserLogger
 
 enum FileType: String {
-  case dat, json, tgz
+  case dat, json
 }
 
 enum AdblockerType {
   case general
-  case httpse
   case regional(locale: String)
-
-  var locale: String? {
-    switch self {
-    case .regional(let locale): return locale
-    default: return nil
-    }
-  }
-
   var associatedFiles: [FileType] { return [.json, fileForStatsLibrary] }
 
   private var fileForStatsLibrary: FileType {
     switch self {
     case .general, .regional: return .dat
-    case .httpse: return .tgz
```

Comment:
here, too. Is it HTTPSEverywhere right? Is it dropped?

Reply:
this endpoint was never used for HTTPSE, we use a static list on iOS 14, and on iOS 15 we use Apple's built in httpse